### DISCUSSION TITLE
Alternative for the OpenTX Adjust Variable Special Function.

### DISCUSSION THREAD

**Discussion by BompaBert** [FrSkyRC/ETHOS-Feedback-Community#3111] [Q&A]
In attached OTX example Capture-1.jpg, you see two Global Variables GV1 and GV3. GV1 represents a differential value for channels CH1/CH5 and GV3 represents a weight for an aileron to rudder mix in channel CH4.
Capture-1 shows related Special Functions SF41 and SF42. Both represent the 'Adjust' Action with as Source the throttle trim. When switch SC- is selected am able to trim the aileron differential in flight and the same goes for the aileron to rudder trim by flipping switch SC up. I also use this setup to adjust Tailerons, Flaperons and Flap to Elevators in both  takeoff and landing routines by using just a few  3-position switches and the throttle trim. I find this very practical especially when flying fast birds such as jets.
In Ethos, is there a way or capability of doing the same?
![Capture-2](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/152647971/8ac80f64-5512-4c31-9fcf-c74069316f1d)
![Capture-1](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/152647971/3acbbe4e-bcdb-4ac0-b88a-c212d7a4637b)

**Comment by arnabraha** [comment]
I have done something similar to adjust the rates and expo on the fly.

1. In the Ailerons mix instead of a value for Differential, use a Channel that you dont use (say CH20)
2. Define two logical switches - Adjust+ and Adjust-. Use AND and use the Switch SC and Trim. This will activate LSW Adjust+ or Adjust- depending on the direction of trim.
3. Go back to Mixer and create a Trim Mix with Up and Down as the LSW defined in 2 and output as CH20.

Hope this helps

**Reply by BompaBert** [reply]
Definitely will help. However, moved back to EdgeTX but will keep your solution in mind.
